bucket/conemu.json

```diff
@@ -38,6 +38,10 @@
         "github": "https://github.com/Maximus5/ConEmu"
     },
     "autoupdate": {
-        "url": "https://github.com/Maximus5/ConEmu/releases/download/v$version/ConEmuPack.$cleanVersion.7z"
+        "url": "https://github.com/Maximus5/ConEmu/releases/download/v$version/ConEmuPack.$cleanVersion.7z",
+        "hash": {
```

Comment:
```suggestion
        "hash": {
            "mode": "fosshub"
        }
```

Reply:
Don't know if it works, just freeze.

```
test: 19.07.14 (scoop version is 19.07.214)
Autoupdating test
```

This freeze, too.

```
        "hash": {
            "url": "https://www.fosshub.com/ConEmu.html",
            "mode": "fosshub"
        }
```
```
test: 19.07.14 (scoop version is 19.037.14)
Autoupdating test
Searching hash for ConEmuPack.190714.7z in https://www.fosshub.com/ConEmu.html
```